snaikArcade.cpp

```diff
@@ -130,12 +128,12 @@ void display()
 	{
 		for(int j=0;j<height;j++)
 		{
-			cout<<ma[i][j];
+			std::cout<<ma[i][j];
 		}
-	cout<<endl;
+	std::cout<<endl;
```

Comment:
The code is not functional unless you replace endl with std::endl
```suggestion
	std::cout<<std::endl;
```

Reply:
@AsociTon You're totally right. I missed that one. My bad.

Reply:
Works dude!